libs/graphics/spirv-headers/monitoring.yml

```diff
@@ -0,0 +1,2 @@
+# No CPE found
+security: {}
```

Comment:
I think the ID is `230542`, however they detect some old tags so I'd add the following exception:
```
releases:
  id: 230542
  ignore:
    # Old tags that predate sdk-* versioning
    - "^1.*"
```